pkg/genlib/generator_interface.go

```diff
@@ -22,6 +22,12 @@ import (
 	"time"
 )
 
+var timeNow time.Time
+
+func init() {
```

Comment:
Is there a way to not use `init` here? I would prefer this to be explicit to reduce unexpected side effects.